crates/frontend/src/pages/experiment_list.rs

```diff
@@ -34,36 +49,334 @@ struct CombinedResource {
+                        <input
+                            type="text"
+                            id="experiment-id-filter"
+                            class="input input-bordered rounded-md resize-y w-full max-w-md"
+                            value=move || filters_rws.get().experiment_ids
+                            placeholder="eg: 7259558160762015744"
+                            on:change=move |event| {
+                                let id = event_target_value(&event);
+                                let id = if id.is_empty() { None } else { Some(id) };
+                                filters_buffer_rws
+                                    .update(|filter| {
+                                        filter.experiment_ids = id;
+                                    });
+                            }
+                        />
+                    </div>
+                    <div class="form-control mt-4">
+                        <label class="label">
+                            <span class="label-text">Created By (Seperate by Comma)</span>
+                        </label>
+                        <input
+                            type="text"
+                            id="experiment-user-filter"
+                            class="input input-bordered rounded-md resize-y w-full max-w-md"
+                            placeholder="eg: user@superposition.io"
+                            value=move || filters_rws.get().created_by
+                            on:change=move |event| {
+                                let user_names = event_target_value(&event);
+                                let user_names = if user_names.is_empty() {
+                                    None
+                                } else {
+                                    Some(user_names)
+                                };
+                                let filters = filters_buffer_rws.get();
+                                let filters = ExperimentListFilters {
+                                    created_by: user_names,
+                                    ..filters
+                                };
+                                filters_buffer_rws.set(filters);
+                            }
+                        />
+
+                    </div>
+                </div>
+                <div class="flex justify-start mt-8">
+                    <Button
+                        class="pl-[70px] pr-[70px] w-48 h-12".to_string()
+                        text="Submit".to_string()
+                        on_click=move |event| {
+                            event.prevent_default();
+                            let filter = filters_buffer_rws.get();
+                            filters_rws.set(filter);
+                            close_drawer("experiment_filter_drawer")
+                        }
+                    />
+                    <Button
+                        class="pl-[70px] pr-[70px] w-48 h-12".to_string()
```

Comment:
can we directly use `px-[70px]` instead
```suggestion
                        class="h-12 w-48 px-[70px]".to_string()
```

Reply:
+1